ctapipe/pipeline/algorithms/prun2ptabhillas.py

```diff
@@ -0,0 +1,37 @@
+from time import sleep
+import threading
+import subprocess
+import os
+from ctapipe.core import Component
+from traitlets import Unicode
+from ctapipe.pipeline.algorithms.base_process import build_command
+
+
+class Prun2PtabHillas(Component):
```

Comment:
rename this class (no idea what it does from the name): 
* don't use abbreviations
*  use consistent capitalization
* Add a `description` field.